src/App.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import axios from "axios";
+import BoardList from "./components/BoardList";
+import CardList from "./components/CardList";
+import BoardForm from "./components/BoardForm";
+import CardForm from "./components/CardForm";
+
+const kBaseUrl = "https://ssh-back-end-inspiration-board.herokuapp.com";
+
+const boardApiToJson = (board) => {
+  const { title, owner, board_id: boardId } = board;
+  return { title, owner, boardId };
+};
+
+const getBoardData = () => {
+  return axios
+    .get(`${kBaseUrl}/boards`)
+    .then((response) => {
+      return response.data.map(boardApiToJson);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+const cardApiToJson = (card) => {
+  const { message, likes_count: likesCount, card_id: cardId } = card;
+  return { message, likesCount, cardId };
+};
+
+const getCardData = (boardId) => {
+  return axios
+    .get(`${kBaseUrl}/boards/${boardId}/cards`)
+    .then((response) => {
+      return response.data.map(cardApiToJson);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+const sortCardData = (unsortedCardData, sortType) => {
+  const unsortedCardDataCopy = [...unsortedCardData];
+
+  if (sortType === "idDown") {
+    const idDescending = unsortedCardDataCopy.sort(
+      (a, b) => b.cardId - a.cardId
+    );
+    return idDescending;
+  } else if (sortType === "idUp") {
+    const idAscending = unsortedCardDataCopy.sort(
+      (a, b) => a.cardId - b.cardId
+    );
+    return idAscending;
+  } else if (sortType === "messageDown") {
+    const msgDescending = unsortedCardDataCopy.sort((a, b) =>
+      a.message > b.message ? -1 : 1
+    );
+    return msgDescending;
+  } else if (sortType === "messageUp") {
+    const msgAscending = unsortedCardDataCopy.sort((a, b) =>
+      a.message > b.message ? 1 : -1
+    );
+    return msgAscending;
+  } else if (sortType === "plusOneDown") {
+    const likesDescending = unsortedCardDataCopy.sort(
+      (a, b) => b.likesCount - a.likesCount
+    );
+    return likesDescending;
+  } else if (sortType === "plusOneUp") {
+    const likesAscending = unsortedCardDataCopy.sort(
+      (a, b) => a.likesCount - b.likesCount
+    );
+    return likesAscending;
+  }
```

Comment:
When you have long if/else trees whose conditions are looking to match something particular, a `switch` statement can be a good replacement: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch